packages/cashscript/test/fixture/mecenas.json

```diff
@@ -17,21 +17,10 @@
   "abi": [
     {
       "name": "receive",
-      "covenant": true,
-      "inputs": [
-        {
-          "name": "pk",
-          "type": "pubkey"
-        },
-        {
-          "name": "s",
-          "type": "sig"
-        }
-      ]
```

Comment:
These aren't needed because the hack is not needed, right?

Reply:
Exactly.